__tests__/app.test.js

```diff
@@ -16,23 +16,57 @@ describe('App', () => {
     describe('GET /api/topics', () => {
         test('should return 404 error if endpoint is misspelled', () => {
             return request(app).get("/api/topic")
-            .expect(404)
+                .expect(404)
         });
         test('should respond with a status of 200 & array of topics objects with properties of slug and description', () => {
             return request(app).get("/api/topics")
-            .expect(200)
-            .then((result) => {
-                const {
-                  body,
-                  body: { topics },
-                } = result;
-                expect(body).toHaveProperty("topics");
-                expect(topics.length).toBeGreaterThanOrEqual(1);
-                topics.forEach((topic) => {
-                  expect(topic).toHaveProperty("slug");
-                  expect(topic).toHaveProperty("description");
+                .expect(200)
+                .then((result) => {
+                    const {
+                        body,
+                        body: { topics },
+                    } = result;
+                    expect(body).toHaveProperty("topics");
+                    expect(topics.length).toBeGreaterThanOrEqual(1);
+                    topics.forEach((topic) => {
+                        expect(topic).toHaveProperty("slug");
+                        expect(topic).toHaveProperty("description");
+                    });
+                });
+        });
+    });
+    describe('GET /api/articles', () => {
+        test('should respond with 404 if incorrect endpoint', () => {
+            return request(app).get("/api/article")
+                .expect(404)
```

Comment:
This shouldn't be part of /api/articles, as it's not specific to that endpoint. Checking for an endpoint that doesn't exist will universally cover your entire app. I'd also expect a message to be returned here to tell the user what's wrong.

Reply:
Actually I've just noticed you already have this as part of /topics. You can get rid of this one and move the topics one outside of the specific endpoint block.